app/navigation/OtherIndividualNavigator.scala

```diff
@@ -70,11 +70,11 @@ class OtherIndividualNavigator @Inject()() extends Navigator {
       yesNoNav(ua, MentalCapacityYesNoPage, navigateToStartDateOrCheckDetails(mode, ua), navigateToStartDateOrCheckDetails(mode, ua))
   }
 
-  private def navigateToPassportDetails(mode: Mode) = {
-    if (mode == NormalMode) {
-      addRts.PassportDetailsYesNoController.onPageLoad(mode)
-    } else {
+  private def navigateToPassportDetails(mode: Mode, ua: UserAnswers) = {
+    if (ua.get(PassportOrIdCardDetailsYesNoPage).isDefined || ua.get(PassportOrIdCardDetailsPage).isDefined) {
       amendRts.PassportOrIdCardDetailsYesNoController.onPageLoad(mode)
+    } else {
+      addRts.PassportDetailsYesNoController.onPageLoad(mode)
     }
   }
```

Comment:
Could this be simplified to something like:

  private def navigateToPassportDetails(mode: Mode, ua:UserAnswers) = {
    (ua.get(PassportOrIdCardDetailsYesNoPage), ua.get(PassportOrIdCardDetailsPage)) match {
      case (Some(_), Some(_)) => amendRts.PassportOrIdCardDetailsYesNoController.onPageLoad(mode)
      case (_, _) => addRts.PassportDetailsYesNoController.onPageLoad(mode)
    }
  }

Reply:
Updated this piece